### ISSUE TITLE
mirror images of cards on some browsers

### ISSUE THREAD

**Issue by moloko** [danielstorey/adapt-flipper#1]
In IE11 and Safari for iOS (iOS 9 and 10), every other item is the mirror image of the previous item:
![2016-10-17 13_01_26-custom components and extensions _ custom components extensions - internet exp](https://cloud.githubusercontent.com/assets/1249597/19436795/195de792-946a-11e6-9a5c-98bdb4c50f7f.png)

Additionally on iOS the scroll had a bit of a tendency to jump around, particularly if I had another flipper component underneath the current one.

**Comment by dominicgisler** [comment]
#3 fixed it. It doesn't look exactly the same as in chrome or firefox, but there is no mirror image anymore.

**Comment by OscarArango** [comment]
Seem like when the mirror issue was fixed, it broke the plugin when used in Chrome.  I had to use the develop version for it to render correctly.  Even with that version, it looks like title and body content is borken.  it shows those items on top of the image even when flipping.
![screen shot 2017-04-06 at 9 08 39 am](https://cloud.githubusercontent.com/assets/5264209/24755531/a789186a-1aa8-11e7-83bb-6b92b3fe11f1.png)
